src/Core/Notifications/INotificationService.cs

```diff
@@ -18,10 +18,89 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System;
+using System.Linq;
+using SonarLint.VisualStudio.Core.InfoBar;
+
 namespace SonarLint.VisualStudio.Core.Notifications
 {
-    public interface INotificationService
+    public interface INotificationService : IDisposable
     {
         void ShowNotification(INotification notification);
     }
+    
+    public sealed class NotificationService : INotificationService
+    {
+        /// <summary>
+        /// Taken from "ToolWindowGuids80.ErrorList"
+        /// </summary>
+        internal static readonly Guid ErrorListToolWindowGuid = new Guid("D78612C7-9962-4B83-95D9-268046DAD23A");
```

Comment:
The error list makes sense for this notification, but in the future I think we'll need to let the caller specify where it should appear.

Reply:
Yep, this is kind of a "temp" solution. I don't want to overcomplicate it now.